lesson_15/lesson_15_Turcu_Alexandru/unittest.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+from lesson_15.lesson_15_Turcu_Alexandru import homework
+
+
+class TestSquareArea(unittest.TestCase):
+    def test_area(self):
+        elem = 2
+        area = elem * elem
```

Comment:
the test idea is good, but you need to make sure you use the code for testing. This test will never fail. You need to use the square class to compute the area


`square = Square(elem,elem,elem,elem)
self.assertEqual(square.area(), area)
`